src/sensai/data_transformation.py

```diff
@@ -395,67 +496,111 @@ def apply(self, df: pd.DataFrame) -> pd.DataFrame:
         self._checkUnhandledColumns(df, matchedRulesByColumn)
         return df
 
+    def summary(self):
```

Comment:
Why are we calling the method `summary`? Wouldn't `info` be more a more obvious choice?

Reply:
I wanted to keep it similar to the keras interface. There models have the summary method. In the future I though of adding it to our models too

Reply:
Why? We are not keras.

Reply:
At least I don't think it's worse than info. If you feel strongly about it, I can change it, of course

Reply:
It really is more of an info than a summary. I changed the name